src/views/FarmAuction/helpers.tsx

```diff
@@ -14,25 +15,15 @@ export const FORM_ADDRESS =
 // Also amends bidder information with getBidderInfo
 // auction is required if data will be used for table display, hence in reclaim and congratulations card its omitted
 export const sortAuctionBidders = (bidders: BidsPerAuction[], auction?: Auction): Bidder[] => {
-  const sortedBidders = [...bidders]
-    .sort((a, b) => {
-      if (a.amount.lt(b.amount)) {
-        return 1
-      }
-      if (a.amount.gt(b.amount)) {
-        return -1
-      }
-      return 0
-    })
-    .map((bidder, index) => {
-      const bidderInfo = getBidderInfo(bidder.account)
-      return {
-        ...bidderInfo,
-        position: index + 1,
-        account: bidder.account,
-        amount: bidder.amount,
-      }
-    })
+  const sortedBidders = orderBy(bidders, (bidder) => bidder.amount, 'desc').map((bidder, index) => {
```

Comment:
@memoyil since `amount` is a BigNumber I think we should cast it `.toNumber()`

Reply:
Done